jdaviz/configs/cubeviz/plugins/tests/test_data_retrieval.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+import numpy as np
+
+from astropy.utils.data import download_file
+
+from jdaviz.app import Application
+
+URL = 'https://stsci.box.com/shared/static/28a88k1qfipo4yxc4p4d40v4axtlal8y.fits'
```

Comment:
Can this be changed to the "pretty" version of the URL? Or alternatively, a comment explaining which file this is?  Right now it's hard to look at this file and understand it.

Relatedly, can we get a quick docstring that says what the scope of this test file is?  I think "retreival" might be interpreted by some readers as being about the download, when this is really about "retrieving from glue" (and a docstring is sufficient to address that!)